CleverSDK/Classes/CLVOAuthManager.m

```diff
@@ -49,12 +49,18 @@ + (void)startWithClientId:(NSString *)clientId clvLoginHandler:(CLVLoginHandler
     manager.clvLogin = clvLoginHandler;
 }
 
-+(NSString*)generateRandomString:(int)num {
-    NSMutableString* string = [NSMutableString stringWithCapacity:32];
-    for (int i = 0; i < num; i++) {
-        [string appendFormat:@"%C", (unichar)('a' + arc4random_uniform(25))];
+
++ (NSString *)generateRandomString:(int)length {
```

Comment:
I doubt it actually matters, but this is unable to generate a string of odd length. It will instead generate a string of length n-1.